Dockerfile.dapper

```diff
@@ -22,8 +22,17 @@ ENV K3S_BINARY_amd64=k3s \
     K3S_BINARY_s390x=k3s-s390x \
     K3S_BINARY=K3S_BINARY_${ARCH}
 
-RUN curl -sL https://github.com/rancher/k3s/releases/download/$(curl -Ls -o /dev/null -w %{url_effective} https://update.k3s.io/v1-release/channels/v1.24 | awk -F/ '{ print $NF }')/${!K3S_BINARY} > /usr/local/bin/k3s && \
-    chmod +x /usr/local/bin/k3s
+ARG PSP_ENABLED
+
+RUN echo "${PSP_ENABLED}"
+
+RUN if [ "${PSP_ENABLED}" == "1" ]; then \
+        curl -sL https://github.com/rancher/k3s/releases/download/$(curl -Ls -o /dev/null -w %{url_effective} https://update.k3s.io/v1-release/channels/v1.24 | awk -F/ '{ print $NF }')/${!K3S_BINARY} > /usr/local/bin/k3s; \
+    else \
+        curl -sL https://github.com/rancher/k3s/releases/download/$(curl -Ls -o /dev/null -w %{url_effective} https://update.k3s.io/v1-release/channels/stable | awk -F/ '{ print $NF }')/${!K3S_BINARY} > /usr/local/bin/k3s; \
```

Comment:
This block is largely the same for both if and else. How do you feel about using a variable for the version, then extracting the `curl`s from if/else?

Reply:
Hmm, that might look cleaner. I'll redo the pr

Reply:
Actually I just remembered, the whole point of that was for people building locally so theres a default value. Either way its gonna be a bit clunky, whether we repeat that line or have a whole if else block just to set the value.